scripts/lib.sh

```diff
@@ -93,7 +95,7 @@ function gw_connect_fed() {
 }
 
 function get_finality_delay() {
-    cat $FM_CFG_DIR/client.json | jq -e -r '.modules.wallet.finality_delay'
```

Comment:
Why remove `-e`? This will likely lead to silent test failures again.

It seems this comes from the "merge master into this branch" commit even though I don't see it in the GH UI?! We need to be careful with these GH UI merge/rebase actions. Better to do this locally and pay attention when both master and feature branch had changes in the same spot.

Reply:
ohh oops. I did this via the UI. Sorry about that.